docs/notes/extensions.md

```diff
@@ -1,26 +1,45 @@
 # Popular Extensions
 
+Many extensions can be found on the [Chart.js GitHub organization](https://github.com/chartjs) or on the [npm registry](https://www.npmjs.com/search?q=chartjs-plugin-).
```

Comment:
The npm link should be changed for `https://www.npmjs.com/search?q=chartjs-` (`plugins-` removed), and we should add back this link at the end of the `##Plugins` section: 

`In addition, many plugins can be found on the [npm registry](https://www.npmjs.com/search?q=chartjs-plugin-).`

We can do the same for charts:

`In addition, many charts can be found on the [npm registry](https://www.npmjs.com/search?q=chartjs-chart-).`